backend/app/routers/flows.py

```
@@ -0,0 +1,235 @@
+        if schema:
+            response_data["schema"] = SchemaResponse.from_orm_model(schema)
+
+    return FlowWithResultResponse(**response_data)
+
+
+@router.get("/flows/{flow_id}/audit", response_model=AuditHistoryResponse)
+async def get_flow_audit_history(
+    flow_id: int,
+    field_path: Optional[str] = Query(None, description="Filter by field path"),
+    limit: int = Query(100, le=500, description="Maximum results"),
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)